website/content/docs/upgrading/upgrade-to-1.15.x.mdx

```diff
@@ -45,6 +45,12 @@ option.
 
 @include 'application-of-sentinel-rgps-via-identity-groups.mdx'
 
+## Auto-roll billing start date
```

Comment:
In this (and the other upgrade guides), does it make more sense to say `auto-rolled` instead of `auto-roll`? Since it's an adjective describing the type of billing start date.